src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -259,6 +259,12 @@ export default defineComponent({
 
     this.updateQueryDebounce = debounce(this.updateQuery, 500)
   },
+  mounted: async function () {
```

Comment:
This function doesn't run any async code and adding an event listener is a quick enough operation that we can do it synchronously.
```suggestion
  mounted: function () {
```

Reply:
This is copied verbatim from another usage; removed it from both instances now.

Reply:
Thank you for removing it in both places.